server_src/utils/env.utils.js

```diff
@@ -20,30 +23,62 @@ function isNode() {
   return "NODE" in process.env;
 }
 
+/**
+ * Turn an object into an envfile string
+ * Copied from https://github.com/bevry/envfile
+ */
+function stringifyDotEnv(obj) {
+  let result = "";
+  for (const [key, value] of Object.entries(obj)) {
+    if (key) {
+      const line = `${key}=${String(value)}`;
+      result += line + "\n";
+    }
+  }
+  return result;
+}
+
+/**
+ * Write a new MONGO value to .env file
+ * Note: assumes in Nodemon, pm2 or PKG mode.
+ */
+function writeVariableToEnvFile(absoluteEnvPath, variableKey, jsonObject) {
```

Comment:
I loved making this:
- it reads the config again, eats it up and merges with new key and its value and writes it to the file again.
- it bails out if the original .env file is bad or erroneous - that's a small point of attention (how to propagate that to user)